src/borg/compress.pyx

```diff
@@ -220,6 +226,35 @@ class ZLIB(CompressorBase):
             raise DecompressionError(str(e)) from None
 
 
+class ZSTD(CompressorBase):
+    """zstd compression / decompression (pypi: zstandard, gh: python-zstandard)"""
+    # This is a NOT THREAD SAFE implementation.
+    # Only ONE python context must to be created at a time.
+    # It should work flawlessly as long as borg will call ONLY ONE compression job at time.
+    ID = b'\x10\x00'
```

Comment:
taken "as is" from @willyvmm's PR.

next free one would be \x03\x00, so maybe we rather use that.